Emby.Naming/Video/VideoListResolver.cs

```diff
@@ -207,5 +261,41 @@ private static bool IsEligibleForMultiVersion(ReadOnlySpan<char> folderName, Rea
                    || testFilename[0] == '-'
                    || CheckMultiVersionRegex().IsMatch(testFilename);
         }
+
+        private static string EpisodeGrouper(string testFilePath, NamingOptions namingOptions, ReadOnlySpan<char> collectionType)
+        {
+            // Grouper for tv shows/episodes should be everything before space-dash-space
+            var resolver = new EpisodeResolver(namingOptions);
+            EpisodeInfo? episodeInfo = resolver.Resolve(testFilePath, false);
```

Comment:
use var

Reply:
did you mean to NOT use `var`?

Reply:
Sorry, I don't get what you mean...

Reply:
```suggestion
            var episodeInfo = resolver.Resolve(testFilePath, false);
```

Reply:
Ah OK :)

I updated my PR accordingly (https://github.com/SenorSmartyPants/jellyfin/pull/121/commits/2e702fb1ac222e3eea10be8d60042a43353df9fe)

Reply:
I updated this PR with that change.